native/src/main/java/io/ballerina/persist/nodegenerator/SyntaxTreeGenerator.java

```diff
@@ -0,0 +1,203 @@
+                                moduleMembers = moduleMembers.add(SampleNodeGenerator.createNumericKV(
+                                        subNode.identifier().toString().trim(),
+                                        defaultValues[indexOf(nodeMap,
+                                                subNode.identifier().toString().trim())], null));
+                            } else {
+                                existingNodes.add(subNode.identifier().toString().trim());
+                                moduleMembers = moduleMembers.add(SampleNodeGenerator.createStringKV(
+                                        subNode.identifier().toString().trim(),
+                                        defaultValues[indexOf(nodeMap,
+                                                subNode.identifier().toString().trim())], null));
+                            }
+                        }
+                    }
+                    if (existingNodes.size() != 5) {
+                        moduleMembers = populateRemaining(moduleMembers, existingNodes);
+                    }
+
+                } else if (!existingNodes.isEmpty() || !moduleMembers.isEmpty()) {
+                    moduleMembers = addNewLine(moduleMembers, 2);
+                    moduleMembers = populateRemaining(moduleMembers, existingNodes);
+                    moduleMembers = addNewLine(moduleMembers, 1);
+                    moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                } else {
+                    moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                }
+            } else if (member instanceof TableArrayNode) {
+                moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+
+            }
+        }
+        if (existingNodes.isEmpty()) {
+            moduleMembers = addNewLine(moduleMembers, 1);
+            moduleMembers = moduleMembers.add(SampleNodeGenerator.createTable(name, null));
+            moduleMembers = populateRemaining(moduleMembers, existingNodes);
+            moduleMembers = addNewLine(moduleMembers, 1);
+
+        }
+        Token eofToken = AbstractNodeFactory.createIdentifierToken("");
+        DocumentNode documentNode = NodeFactory.createDocumentNode(moduleMembers, eofToken);
+        TextDocument textDocument = TextDocuments.from(documentNode.toSourceCode());
+        return SyntaxTree.from(textDocument);
+    }
+
+    private static boolean isDatabaseConfigurationEntry(KeyNode key) {
+        switch (key.toString().trim()) {
+            case KEY_USER:
+            case KEY_DATABASE:
+            case KEY_PASSWORD:
+            case KEY_HOST:
+            case KEY_PORT:
+                return true;
+            default:
+                return false;
+        }
+    }
+    private static int indexOf(String[] arr, String key) {
+        int index = 0;
+        for (String member : arr) {
```

Comment:
```suggestion
        private static int indexOf(String key) {
        int index = 0;
        for (String member : SyntaxTreeGenerator.nodeMap) {
```

Reply:
We can directly use `SyntaxTreeGenerator.nodeMap` instead of `arr`. Update the usages of this function.